Weather/Views/TimedForecastCellView.swift

```diff
@@ -9,21 +9,62 @@
 import UIKit
 
 class TimedForecastCellView: UICollectionViewCell {
```

Comment:
As I said in my opinion subclassing UIViews is not a good idea so consider marking this class as `final class`.
This will help not to make subclass without really thinking of what you are doing. Also helps another developers not to do that if there is no real need.